### ISSUE TITLE
Failed to build TensorRT engine because the weights are outside f16 range.

### ISSUE THREAD

**Issue by jrnladhad** [jkjung-avt/tensorrt_demos#327]
Thank you for such a useful tool. I have a custom YOLOv3 model that I trained using weights from the official website and then fine-tuned them to adjust to my data and network architecture using PyTorch. Once I obtained the new weights I was able to generate the onnx format using the script provided by you but fail to create the tensorRT engine. 

Using Jetson Xavier NX to convert the model.

ERROR: ../builder/cudnnBuilderWeightConverters.cpp (541) - Misc Error in operator(): 1 (Weights are outside of fp16 range. A possible fix is to retrain the model with regularization to bring the magnitude of the weights down.)

**Comment by jkjung-avt** [comment]
https://github.com/jkjung-avt/tensorrt_demos/issues/161#issuecomment-662473343

**Comment by jrnladhad** [comment]
Hey, 
I retrained the model and lowered the decay value during the training, earlier it was 0.0005 and this time it was 0.005 and yet I am running into the same issue.

**Comment by jkjung-avt** [comment]
You could try to lower learning rate (while keeping the same weight decay value) and train the model for a longer period of time.

If that still doesn't work, then you should probably consider using FP32 TensorRT engine.
